pkgs/applications/misc/grsync/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, dee, gtk2, intltool, libdbusmenu-gtk2, libunity, pkg-config, rsync }:
+
+stdenv.mkDerivation rec {
+  version = "1.2.6";
+  pname = "grsync";
+
+  src = fetchurl {
+    url = "https://kent.dl.sourceforge.net/project/grsync/grsync-${version}.tar.gz";
+    sha256 = "06ani65d58p8r3jvxjwpwyqrr07ya3icdqc243nxcrv7bvmarmb6";
+  };
+
+  buildInputs = [
+    dee
+    gtk2
+    intltool
+    libdbusmenu-gtk2
+    libunity
+    pkg-config
```

Comment:
pkg-config and possibly intltool should be nativeBuildInputs (executed during build not at runtime)

Reply:
Fixed